addon/instance-initializers/firebase-settings.ts

```diff
@@ -1,6 +1,7 @@
 import ApplicationInstance from '@ember/application/instance';
+import { getAuth, connectAuthEmulator } from 'firebase/auth';
 
-import firebase from 'firebase/compat/app';
+import type firebase from 'firebase/compat/app';
```

Comment:
This is interesting, first time I've seen `import type`. This would've been a nice solution for #190 😅